### PR TITLE
docs: Add JSDoc comments to Page component

### PR DESCRIPTION
**Pull request by dulcehc**
**Thank you for your contribution!** 👏

To get it merged faster, kindly review the checklist below:

## Pull Request Checklist
- [ X] Reviewed the [Contributing Guidelines](https://github.com/SAP/ui5-webcomponents-react/blob/master/CONTRIBUTING.md)
    + Especially the [How to Contribute](https://github.com/SAP/ui5-webcomponents-react/blob/master/CONTRIBUTING.md#contribute-code) section 
- [ X] [Correct commit message style](https://github.com/SAP/ui5-webcomponents-react/blob/master/docs/Guidelines.md#commit-message-style)

### CODE REVIEW

File: packages/main/src/components/Page/index.tsx [hunk 1]
```diff
@@ -14,21 +14,52 @@ import { CommonProps } from '../../interfaces/CommonProps';
...
+  /**
+   * The title text appearing in the page header bar.
+   */
   title?: string;
+  /**
+   * It is used to set the background color of a page.
+   */
   backgroundDesign?: PageBackgroundDesign;
+  /**
+   * The (optional) custom header of this page. Use this aggregation only when a custom header
```
Comment (MarcusNotheis) [inline packages/main/src/components/Page/index.tsx:26]:
```suggestion
   * The (optional) custom header of this page. Use this prop only when a custom header
```
`aggregation` is in my opinion very tightly coupled to UI5, I would prefer `prop` here is it's more the "react-way"

File: packages/main/src/components/Page/index.tsx [hunk 2]
```diff
@@ -14,21 +14,52 @@ import { CommonProps } from '../../interfaces/CommonProps';
...
+  /**
+   * Whether this page shall have a header.
+   */
   showHeader?: boolean;
+  /**
+   * This event is fired when Nav Button is pressed.
+   */
   onNavButtonPress?: (e: CustomEvent<{}>) => void;
+  /**
+   * The page has three distinct areas that can hold content - a header, content area and a footer.
```
Comment (MarcusNotheis) [inline packages/main/src/components/Page/index.tsx:51]:
```suggestion
   * The content area of the page.
```
footer and header are separate props, so content area is the only which is left here

File: packages/main/src/components/Page/index.tsx [hunk 3]
```diff
@@ -14,21 +14,52 @@ import { CommonProps } from '../../interfaces/CommonProps';
...
+   */
   children: ReactElement<any> | ReactElement<any>[] | ReactNode;
 }
 
 const useStyles = createComponentStyles(styles, {
   name: 'Page'
 });
 
+/**
+ * A container control that holds one whole screen of an application.
```
Comment (MarcusNotheis) [inline packages/main/src/components/Page/index.tsx:61]:
```suggestion
 * A layout component that holds one whole screen of an application.
```
I think layout component is also more the "react-way" of describing it.

**Review by MarcusNotheis — CHANGES_REQUESTED** [review]

**Comment by dulcehc** [comment]
thanks @MarcusNotheis for your review, I have updated the PR

**Review by MarcusNotheis — APPROVED** [review]

**Pull request merged**